features/lockscreen/impl/src/main/kotlin/io/element/android/features/lockscreen/impl/storage/LockScreenStore.kt

```diff
@@ -32,6 +34,14 @@ interface PinCodeStore : EncryptedPinCodeStorage {
      * Resets the counter of attempts for PIN code and biometric access.
      */
     suspend fun resetCounter()
-}
 
+    /**
+     * Returns whether the biometric unlock is allowed or not.
+     */
+    fun isBiometricUnlockAllowed(): Flow<Boolean>
```

Comment:
Nit: maybe use `enabled` here instead of `allowed`? Allowed sounds like something coming from the OS to me, rather than from a user preference.

Reply:
It was to reflects what we show on the ui...